src/components/Datasets.js

```diff
@@ -38,6 +38,10 @@ const styles = (theme) => ({
   infoAlert: {
     margin: theme.spacing(2),
   },
+  content: {
+    // adds bottom space so that button doesn't stay above table when fully scrolled
+    marginBottom: theme.spacing(10),
```

Comment:
I approve this change, it improves the user experience. However, shouldn't it be added to the table element instead?

Reply:
I don't think it should be in the table element. The margin is only there because of the add button and not all the pages have have it. The accommodated space amount might also be bigger or smaller if the page has another fixed component. So I think the component using the table is the one that should handle the space issues.